packages/terra-form-input/tests/wdio/form-input-spec.js

```diff
@@ -71,6 +71,42 @@ describe('Form-Input', () => {
       });
     });
 
+    describe('Enabled Placeholder', () => {
+      beforeEach(() => {
+        browser.url('/#/raw/tests/terra-form-input/form-input/placeholder-form-input');
+      });
+
+      Terra.should.beAccessible({ viewports });
+      Terra.should.matchScreenshot({ viewports });
+      Terra.should.themeEachCustomProperty({
+        '--terra-form-input-placeholder-color': 'red',
```

Comment:
Let's add `'--terra-form-input-placeholder-font-style': 'italic',` and then we can remove the placeholder-form-input-font-styled page.